gtfs-realtime/CHANGES.md

```diff
@@ -0,0 +1,80 @@
+The official specification, reference and documentation are written in English. If a translation to a different language differs from the English original, the latter takes precedence. All communication is performed in English.
+
+1. Create a git branch with update of all relevant parts of protocol definition, specification and documentation files (except for translations).
+1. Create pull request on https://github.com/google/transit. Pull request must contain an extended description of the patch. The creator of the pull request becomes the _advocate_.
+1. Once pull request is registered, it must be announced by its advocate in the [GTFS Realtime mailing list](https://groups.google.com/forum/#!forum/gtfs-realtime). Once announced, the pull request is considered a proposal.
+  - Advocate is a contributor, thus they should sign [CLA](../CONTRIBUTING.md) before pull request can be accepted.
+1. The discussion of the proposal follows. Pull request comments should be used as the sole discussion forum.
+  - The discussion lasts for as long as the advocate considers required but at least 7 calendar days.
+  - The advocate is responsible for timely update of the proposal based on the comments for which they agree to.
+  - At any point in time the advocate can claim proposal abandoned.
+1. The advocate can call for a vote on a version of the proposal at any point in time but no earlier than 7 calendar days, and no later than 30 calendar days after the proposal was announced.
+1. Vote lasts the minimum period enough to cover 7 full calendar days and at 5 full Swiss business days. Vote ends at 23:59 UTC.
+  - The advocate should announce the specific end time at the start of the vote.
+  - During voting period only editorial changes to the proposal are allowed (typos, wording may change as long as it does not change the meaning).
+  - Anyone is allowed to vote yes/no in a form of comment to the pull request, and votes can be changed until the end of the voting period.
+    If a voter changes her vote, it is recommended to do it by updating the original vote comment by striking through the vote and writing the new vote.
+  - Votes before the start of the voting period are not considered.
+1. The proposal is accepted if there is a consensus yes with at least 3 votes.
+  - Votes against shall be motivated, and ideally provide actionable feedback.
+  - If the vote has failed, then the advocate may choose to continue work on the proposal, or to abandon the proposal.
+    Either decision of the advocate must be announced in the mailing list.
+  - If the advocate continues the work on proposal then a new vote can be called for at any point in time but no later than 30 calendar days after the end of the previous vote.
+  - If a vote was not called within 30 calendar days from the original proposal or 30 calendar days since end of the previous vote, then the proposal is abandoned.
+  - If 3 votes have failed, then the proposal is abandoned.
```

Comment:
@magdalar I dropped this line. My original intent was to not have dangling proposals, and close them in a timely manner, however this intent is well covered by the above "30 days silence = abandoned" rule.